tests/Benchmarks/Benchmarks.csproj

```diff
@@ -20,10 +21,10 @@
     <NoWarn>$(NoWarn);CS0618</NoWarn>
   </PropertyGroup>
   <ItemGroup>
+    <PackageReference Include="BenchmarkDotNet" Version="0.10.14.525" />
+    <PackageReference Include="BenchmarkDotNet.Diagnostics.Windows" Version="0.10.14.525" />
```

Comment:
Can the benchmarks be run on Linux/Mac?

Reply:
yes, `BenchmarkDotNet.Diagnostics.Windows` is optional and will be used only for HardwareCounters and InliningDiagnoser. It targets .NET Standard 2.0 so it will build on any OS